### PR TITLE
Feature/minio config

### PR DESCRIPTION
**Pull request by kafkaphoenix**
## Description

add minio config to app yaml

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [ ] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: process-trigger/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The configuration should be defined inside the `centralized configuration scope`
 | version | no        | str  | Subject to subscribe     |
```
Comment (alexrodfe) [inline process-trigger/README.md:23]: too dull of an explanation, I'd write something more in the lines of:
version: "the version in which is the process you want to subscribe to"

File: process-trigger/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The configuration should be defined inside the `centralized configuration scope`
...
 | process | no        | str  | Subject to subscribe     |
 | workflow | no        | str  | Subject to subscribe     |
-| message | yes        | bool  | Default value is true      |
+| retain-execution-id | yes        | bool  | Default value is true      |
```
Comment (alexrodfe) [inline process-trigger/README.md:26]: default value is true but what does that mean

File: process-trigger/README.md (hunk 1 above)
Comment (kafkaphoenix) [reply process-trigger/README.md:23]: good idea updated!

**Review by dfinteligenz — APPROVED** [review]

**Pull request merged**